iac-common/src/test/java/org/sonar/iac/common/extension/ParseExceptionTest.java

```diff
@@ -33,16 +35,18 @@ class ParseExceptionTest {
   private final RuntimeException cause = new RuntimeException("cause");
   private final TextPointer position = new BasicTextPointer(1, 2);
   private InputFile inputFile;
+  private InputFileContext inputFileContext;
 
   @BeforeEach
   public void init() {
     inputFile = mock(InputFile.class);
     when(inputFile.toString()).thenReturn("dir1/dir2/TestFile.abc");
+    inputFileContext = new InputFileContext(null, inputFile);
   }
 
   @Test
-  void shouldCreateException() {
-    ParseException actual = ParseException.throwParseException("action", inputFile, cause, position);
+  void shouldCreateGeneralException() {
+    ParseException actual = ParseException.throwGeneralParseException("action", inputFile, cause, position);
```

Comment:
Suggestion: static import of this method.
Same suggestions for other calls in others files.